crates/matrix-sdk-crypto/src/file_encryption/key_export.rs

```diff
@@ -319,7 +321,8 @@ mod test {
         let encrypted = encrypt_key_export(&export, "1234", 1).unwrap();
         let decrypted = decrypt_key_export(Cursor::new(encrypted), "1234").unwrap();
 
-        assert_eq!(export, decrypted);
+        // TODO, what should we do about PartialEq here?
```

Comment:
We used to rely on the fact that libolm exports a room key (or inbound group session) as a string, which we could compare using `PartialEq`.

Vodozemac exports instead a struct that wraps the string to disallow `PartialEq` between secret values. This in turn makes it harder to check if our export mechanism works correctly. We still can get to the actual string using `as_str()`, so doing that instead of adding a `PartialEq`. Fixed in: https://github.com/matrix-org/matrix-rust-sdk/pull/494/commits/a7aef38d384354af35d87de6e613d339ff7a5f89